app/panel/components/Summary.jsx

```diff
@@ -752,34 +752,34 @@ class Summary extends React.Component {
 	 * @return {JSX} JSX for rendering the plus upgrade banner or subscriber icon
 	 */
 	_renderPlusUpgradeBannerOrSubscriberIcon() {
-		const { is_expert, current_theme } = this.props;
+		const { is_expert, current_theme, user } = this.props;
 
-		const isPremiumSubscriber = this._isPremiumSubscriber();
-		const isPlusSubscriber = this._isPlusSubscriber();
+		const hasPremiumAccess = user && user.premiumAccess;
+		const hasPlusAccess = user && user.plusAccess;
```

Comment:
Man I'm SO ready for optional chaining to be a thing

Reply:
Ooh, I've never heard of optional chaining. Looks interesting